crates/jxl-grid/src/test/subgrids.rs

```diff
@@ -1,4 +1,8 @@
+#[cfg(all(target_arch = "aarch64"))]
+use std::arch::aarch64::float32x4_t;
```

Comment:
`cargo fmt` complains about this; I think adding a blank line after this will do the thing.